### PR TITLE
add details on env variables to api tutorials

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/camunda-docs/issues/3915.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

**Will backport upon approval.**

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/apis-tools/administration-api/tutorial.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ To set up your credentials, create an `.env` file which will be protected by the
 
 These keys will be consumed by the `auth.js` file to execute the OAuth protocol, and should be saved when you generate your client credentials in [prerequisites](#prerequisites).
 
+:::note Can't find your environment variables?
```
Comment (pepopowitz) [inline docs/apis-tools/administration-api/tutorial.md:36]:
```suggestion
:::tip Can't find your environment variables?
```

non-blocking: Are these blocks more of a tip than a note?

Comment (christinaausley) [reply docs/apis-tools/administration-api/tutorial.md]: Yes I agree!

**Comment by christinaausley** [comment]
Backported and ready for final review @pepopowitz 👍

**Review by pepopowitz — APPROVED** [review]
🙌

**Pull request merged**